### DISCUSSION TITLE
InsertInto 导致被插入表主键断层

### DISCUSSION THREAD

**Discussion by CharlesFangKang** [dotnetcore/FreeSql#1395] [Q&A]
各位大神
在使用下面方法批量插入时出现 被插入表自增字段出现断层 
被插入表主键从101~105开始缺失，是不是InsertInto不支持MySql？
var resulr = freeSql.Select<Topic1>().Limit(10).InsertInto(null, a => new Topic2 { Clicks = a.Clicks, Title = a.Title, CreateTime = a.CreateTime });

91 92 93 94 95 96 97 98 99 100
106 107 108 109 110 111 112 113 114 115

**Comment by densen2014** [comment]
看执行语句，输出一下sql丢navicat执行一下。按道理不关freesql的事情
